doc/conf.py

```diff
@@ -241,7 +246,32 @@
 #html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-#html_sidebars = {}
+html_sidebars = {
+    "**": [
+        "sidebar/brand.html",
+        "sidebar/search.html",
+        "sidebar/navigation.html",
+        "sidebar/version_toggler.html",  # version toggler template
```

Comment:
@Badr-MOUFAD maybe move it above as it contains important information? otherwise looking great!

Reply:
Doing so, +small adjustments, we get this render

<img alt="screenshot for render" src="https://github.com/scikit-learn-contrib/skglm/assets/65614794/d0fa7a51-85e2-439e-af55-0e749646594f" height="400px"/>

Looks great @mathurinm, :+1: for this change